lnbits/wallets/lnbits.py

```diff
@@ -121,10 +121,12 @@ async def get_invoice_status(self, checking_id: str) -> PaymentStatus:
             return PaymentStatus(None)
 
     async def get_payment_status(self, checking_id: str) -> PaymentStatus:
-        r = await self.client.get(url=f"/api/v1/payments/{checking_id}")
+        try:
+            r = await self.client.get(url=f"/api/v1/payments/{checking_id}")
+            r.raise_for_status()
+        except Exception:
+            return PaymentStatus(None)
```

Comment:
CI fails - shouldn't this be

```suggestion
            return PaymentStatus(False)
```

?

Reply:
Nope, that's the bug I'm fixing here.